python/lsst/ts/standardscripts/maintel/laser_tracker/setup.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["SetUp"]
+
+import asyncio
+import logging
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel import MTCS
+
+
+class SetUp(salobj.BaseScript):
+    """
+    Set Up Laser Tracker component.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    None
+
+    """
+
+    def __init__(self, index):
+        super().__init__(index, descr="Set up laser tracker.")
+
+        self.config = None
+
+        self.laser_tracker = salobj.Remote(self.domain, name="LaserTracker", index=1)
+        self.mtcs = MTCS(domain=self.domain, log=self.log)
+
+        self.log.setLevel(logging.DEBUG)
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        self.config = config
+
+    async def startup(self):
+        await self.laser_tracker.start_task(timeout=5)
```

Comment:
`start_task` is an task not a coroutine.. you should do:

```
await self.laser_tracker.start_task
```

Since you are awaiting 2 tasks, you can do that concurrently with:

```
await asyncio.gather(
    self.laser_tracker.start_task,
    self.mtcs.start_task,
)
```

you can also add a timeout to this, but it usually is not necessary.

Reply:
Removed mtcs, so no longer needed